docs/topics/maven-access-reports.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * docs/maven-guide/master.adoc
 
+:_content-type: REFERENCE
 [id="maven-access-reports_{context}"]
 = Accessing the report
```

Comment:
@stoobie @sbeskin-redhat This is a strange module to start with.  The title suggests it is a procedure but it reads like a verification step. It follows two procedure modules, each of which describes how to run Maven and generate a report under different circumstances.  For now, I'd suggest labeling it a procedure because it describes an action all users should do, not just background information. Another possibility is changing the title to something like "Contents of the Output report directory" and marking it as a concept module.

Reply:
Will open a separate bug for editing the module's content